src/main/java/seedu/nutrigoals/model/Calorie.java

```diff
@@ -49,6 +49,32 @@ public int getCalorieValue() {
         return Integer.parseInt(value);
     }
 
+    /**
+     * Calculates the caloric difference
+     * @param other Calorie to subtract
+     * @return Caloric difference
+     */
+    public int calculateDifference(Calorie other) {
```

Comment:
Might be good to have some test cases for these new methods added!

Reply:
Added!